espnet/asr/chainer_backend/asr.py

```diff
@@ -187,7 +190,10 @@ def __call__(self, batch, device):
 
 
 def train(args):
-    '''Run training'''
+    """Train with the given args
+
+    :param namespace args: The program arguments
```

Comment:
I think this is `Namespace` https://docs.python.org/3.6/library/argparse.html#argparse.Namespace